.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Generate release
+
+on:
+  push:
+    tags:
+      - "v*"
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "v18.16.0"
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
does this match the python-version in the AG Grid tests?

Reply:
The tests run on 3.8-3.12 inclusive: Because of this we expect a consistent build on any version and can increment this with some regular cadence.